src/main/java/org/spongepowered/api/command/spec/CommandExecutor.java

```diff
@@ -41,5 +41,5 @@
      * @return the result of executing this command
      * @throws CommandException If a user-facing error occurs while executing this command
      */
-    CommandResult execute(CommandSource src, CommandContext args) throws CommandException;
+    CommandResult execute(T src, CommandContext args) throws CommandException;
```

Comment:
Rename `src` to `source`? It will match args in other command-related ifaces/classes.